demisto_sdk/commands/pre_commit/hooks/sourcery.py

```diff
@@ -0,0 +1,18 @@
+from pathlib import Path
+
+from demisto_sdk.commands.common import tools
+from demisto_sdk.commands.pre_commit.hooks.hook import Hook
+
+
+class SourceryHook(Hook):
+    def prepare_hook(self, python_version: str, config_file_path: Path, **kwargs):
+        config_file = tools.get_file_or_remote(config_file_path)
+        config_file["rule_settings"]["python_version"] = python_version
+        tmp_file_path = config_file_path.with_name(".sourcery_tmp.yaml")
```

Comment:
Im pretty sure a tmp file might die by the time we actually use it, thats why we do it like that.

Reply:
the temp file folder won't last beyond this context. 
i think it might be simpler to add to the Hook class, a post_run action function for cleanup

Reply:
I prefer a tempfile with `delete=False` (created in a some deep `/temp/...` path nobody will ever see) over writing files into the users' `content` folders that will just stay there and confuse them.

Reply:
@dorschw you have a high chance that this will will be deleted before it's being used.
Either do what i suggested with a post-hook. Or pass the temp directory to the hooks when calling them.

Reply:
i'm for not creating the file in the content directory

Reply:
it won't be deleted (not by python, at least) if passing `delete=False`